client/components/action-panel/figure.jsx

```diff
@@ -8,10 +8,11 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
-const ActionPanelFigure = ( { inlineBodyText, children } ) => {
+const ActionPanelFigure = ( { inlineBodyText, align, children } ) => {
 	const figureClasses = classNames( {
 		'action-panel__figure': true,
 		'is-inline-body-text': inlineBodyText,
+		'is-aligned-left': 'left' === align,
```

Comment:
How does this work with right-to-left languages? Seems like we'll end up with a `is-aligned-left` class with a `float:right`. Is that expected? Seems like it could be confusing.

Also I wonder if it makes more sense to add a `align-${ align }` classname instead to make this more robust.

Reply:
Yep, that doesn't seem best semantically. Perhaps we can have an `align-first` and `align-last` or `align-before` and `align-after`?

Reply:
No, don't get me wrong, I'm fine with it being `left` and `right`. "Before" and "after" might make it more confusing, at least in my head. What I meant we could use `align-${ align }` and add classes to style it properly. 

But also, I'm concerned about how things would behave if we hardcode alignment to be "left" and then we switch to an RTL language. Let's consider that too.

Reply:
> But also, I'm concerned about how things would behave if we hardcode alignment to be "left" and then we switch to an RTL language. Let's consider that too.

Yes, that was why I was wondering if we shouldn't be using 'left' and 'right' in the class names. If something is on the left and we switch to an RTL language, would we want that element to be placed on the right? If so, we'd have a situation where `align-left` is styled to align something on the right, which seems pretty confusing and what it seemed like you were getting at with your original comment. 

I'll investigate what happens currently with RTL languages, but I think the figures will stay on the right, so it may make sense to leave them where they are for consistency.